File: Estrutura_Repetição/C#/Ex1064/Main.cs
using System;

class MainClass {
    static void Main(string[] args) {
        int positivos = 0;
        float valor = 0;
        float media = 0;

        for (int i = 0; i < 6; i++) {
            valor = float.Parse(Console.ReadLine());
            if (valor > 0) {
                positivos++;
                media += valor;
            }
        }

        Console.WriteLine($"{positivos} valores positivos");
        Console.WriteLine($"{media / positivos:F1}");
    }
}

File: Estrutura_Repetição/C/Ex1064/Main.c
#include <stdio.h>

int main() {
    int positivos = 0;
    float valor = 0.0;
    float media = 0.0;
    for (int i = 0; i <= 5; i++) {
        scanf("%f", &valor);
        if (valor > 0) {
            positivos += 1;
            media += valor;
        }
    }
    printf("%d valores positivos\n", positivos);
    printf("%.1f\n", media / positivos);

    return 0;
}